base_layer/common_types/src/types/bullet_rangeproofs.rs

```diff
@@ -32,14 +31,18 @@ use serde::{
 };
 use tari_utilities::{hex::*, ByteArray, ByteArrayError, Hashable};
 
-use crate::types::Blake256;
+use super::{base_layer_common_types_domain_hasher, BULLET_RANGEPROOFS_HASHER_LABEL};
```

Comment:
As per simplification:
```suggestion
use crate::types::BulletRangeproofsHasherBlake256;
```